docs/dev/framework/caching.md

```diff
@@ -275,6 +275,12 @@ Moreover, you don't have to register to all the different callbacks such as `ons
 You can register to the [`oninvalidate_cache_tags` callback][5] and add your own tags.
 
 
+{{% notice tip %}}
```

Comment:
```suggestion
{{% notice "tip" %}}
```

Have you tested whether this works also without the quotes? I also noticed this in other PRs.